cognite_toolkit/_cdf_tk/commands/auth.py

```diff
@@ -119,17 +119,18 @@ def check_auth(
             raise AuthorizationError(
                 "Not a valid authentication token. Check credentials (CDF_CLIENT_ID/CDF_CLIENT_SECRET or CDF_TOKEN)."
             )
-        try:
-            print("Checking projects that the service principal/application has access to...")
-            if len(resp.projects) == 0:
-                raise AuthorizationError(
-                    "The service principal/application configured for this client does not have access to any projects."
-                )
-            projects = ""
-            projects = projects.join(f"  - {p.url_name}\n" for p in resp.projects)
-            print(projects[0:-1])
-        except Exception as e:
```

Comment:
This try .. except block is bad as the only exception that can happen is the AuthorizationError which will be caught and replaced by a worse more generic error message.